Makefile

```diff
@@ -119,8 +119,8 @@ ko:
 		github.com/sigstore/rekor/cmd/rekor-cli
 
 sign-container: ko
-	cosign sign -key .github/workflows/cosign.key -a GIT_HASH=$(GIT_HASH) ${KO_DOCKER_REPO}/rekor-server:$(GIT_HASH)
-	cosign sign -key .github/workflows/cosign.key -a GIT_HASH=$(GIT_HASH) ${KO_DOCKER_REPO}/rekor-cli:$(GIT_HASH)
+	cosign sign --force -a GIT_HASH=$(GIT_HASH) ${KO_DOCKER_REPO}/rekor-server:$(GIT_HASH)
+	cosign sign --force -a GIT_HASH=$(GIT_HASH) ${KO_DOCKER_REPO}/rekor-cli:$(GIT_HASH)
```

Comment:
We will need to revert these lines , as keyless won't work outside of a github action with `id-token: write` or having someone to manage a browser session. 

I guess the logic here is that ko builds rekor and then pushes to gcr.io. If we could do that outside of ko in a GH action then we can get rid of stashing private keys into the repo (which I swear one day is going to turn around a bite us).

Reply:
sounds good we can use the target that I've added here https://github.com/sigstore/rekor/pull/533/files#diff-1fd8c247611fff773b4bbd9d20b2e30557185c2893bb6613404996c3c2290816R35 to sign using keyless

Reply:
reverted

Reply:
added a target for keyless/ci let me know what you think about that